srcpkgs/cross-i686-pc-linux-gnu/files/bug90756.patch

```diff
@@ -0,0 +1 @@
+../../gcc/patches/bug90756.patch
```

Comment:
Same here. Add `_apply_patch -p0 ${FILESDIR}/bug90756.patch` in `_gcc_bootstrap()`.